google/cloud/bigtable/README.md

```diff
@@ -42,7 +42,11 @@ Apache 2.0; see [`LICENSE`](../../../LICENSE) for details.
 
 ## Release Notes
 
-### v1.3.x - TBD
+### v1.4.x - TBD
+
+### v1.3.x - 2019-10
```

Comment:
nit: I would just duplicate the other release note, but I also do no think it is that important.